src/logic/community/communities.ts

```diff
@@ -248,11 +287,19 @@ export function createCommunityComponent(
         await communityPlaces.validateOwnership(placeIds, community.ownerAddress)
       }
 
+      const { needsManualReview, reason: manualReviewReason } =
```

Comment:
Sorry if we have talked about this, but aren't we delaying the process about approving communities manually? if that's the case also for creations, we should return a 400 status code instead of marking it as pending to be reviewed manually